NotesBackEnd/src/Notes.Api/Notes.Api.csproj

```diff
@@ -159,6 +161,10 @@
     <Folder Include="App_Data\" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\Notes.Api.Services\Notes.Api.Services.csproj">

```

Comment:
Why are you including all these references? It does not make sense regards the commit message and the change above…